r/tests/testthat/test-Array.R

```diff
@@ -230,13 +230,13 @@ test_that("array supports Date (ARROW-3340)", {
 test_that("array supports POSIXct (ARROW-3340)", {
   times <- lubridate::ymd_hms("2018-10-07 19:04:05") + 1:10
   a <- array(times)
-  expect_equal(a$type(), date64())
+  expect_equal(a$type(), timestamp(unit = TimeUnit$NANO))
```

Comment:
Since `timestamp()` does not define a default: https://github.com/apache/arrow/blob/master/r/R/R6.R#L319. Or, maybe I'm misunderstanding what you mean by native?

Reply:
Why are you converting the representation of `times` to nanoseconds? I thought POSIXct was represented internally as seconds